README.md

```diff
@@ -226,6 +226,41 @@ Follow Helm Chart installation instructions [here](https://docs.openshift.com/co
 
 ## Additional information
 
+### Additional Workflow Namespaces
+
+If you want to deploy workflows on other namespaces, make sure you follow these steps:
+
+ - For the cluster Sonataflow services to accept the traffic from the workflows, set this label:
+   ```console
+      oc label ns my-additional-workflow-ns rhdh.redhat.com/workflow-namespace=""
```

Comment:
to avoid blindly copy&paste, do we want to specify that `my-additional-workflow-ns` it's customizable?

Reply:
Sorry can you give me an example of what you mean?